include/vmm_arch.h

```diff
@@ -474,6 +474,7 @@ typedef uint8_t cache_type_t;
 #define MSR_APIC_BASE              (0x01B)
 #define MSR_FEATURE_CONTROL        ((uint32_t)0x03A)
 #define MSR_TSC_ADJUST             ((uint32_t)0x03B)
+#define MSR_BIOS_UPDT_TRIG         ((uint32_t)0x079)
```

Comment:
also remove the "MSR_BIOS_UPDT_IRIG" in vmenter_check.c and use the definition here.